public-ui/src/app/views/quick-search/quick-search.component.html

```diff
@@ -35,7 +35,7 @@ <h1 id='db-title' class="primary-display ">Data Browser</h1>
     <div class="iconlinks">
       <div class="icons"><a href="#" alt="FAQs"><img src="/assets/icons/icons_faqs.png" /><span>FAQs</span></a>
       </div>
-      <div class="icons"><a href="#" alt="Introductory Videos"><img src="/assets/icons/icons_introductoryvideo.png" /><span>Introductory Video</span></a>
+      <div class="icons"><a [routerLink]="['introductory-videos']" href="/introductory-videos" alt="Introductory Videos"><img src="/assets/icons/icons_introductoryvideo.png" /><span>Introductory Videos</span></a>
```

Comment:
I think the `href="/introductory-videos"` is redundant since the routerLink should generate it